### DISCUSSION TITLE
Course feedback on "Hello GitHub Actions"

### DISCUSSION THREAD

**Discussion by hopealiang** [skills/.github#776] [Hello GitHub Actions]
This course was great at the beginning, I would add more details on how to navigate to the file for beginners that are just now learning how to use GitHub instead of saying "navigate to the file." The Second thing, the page never reloaded after waiting for 20secs and hitting the refresh button. This left you stuck at step 10 without being able to advance.

**Comment by aflacake** [comment]
Different visibility repository change private, public-repo is better
